manifest.yml.j2

```
@@ -149,6 +149,8 @@ applications:
 
       DVLA_EMAIL_ADDRESSES: '{{ DVLA_EMAIL_ADDRESSES | tojson  }}'
 
+      NEW_RELIC_LICENSE_KEY: '{{ NEW_RELIC_LICENSE_KEY }}'
```

could define this rather than relying on manually updating every line in paas_app_wrapper.sh

```suggestion
      NEW_RELIC_LICENSE_KEY: '{{ NEW_RELIC_LICENSE_KEY }}'
      NEW_RELIC_APP_NAME: '{{ CF_APP }}'
```

there's probably some thinking to do on if we care about having `notify` at the beginning (`notify-api`, `notify-delivery-worker-sender`, etc). i'd probably say yes so it's easier to match up with gunicorn, etc (and we've already defined the admin app as `notify-admin`, but it might just be boiler plate which makes it faffy to search on newrelic 🤔

---

Have done this. We have to rip out `app_name` from newrelic.ini as the config file overrides environment variables (https://docs.newrelic.com/docs/apm/agents/python-agent/configuration/python-agent-configuration/#environment-variables)